apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -108,8 +115,8 @@ export default class DetailViewContents extends React.Component {
             style={{marginRight: '5px'}}
           >
             {
-              STATUSES.map((status, i) => (
-                <option value={status.toLowerCase().replace(/ /g, '_')} key={i}>
+              this.statuses().map((status, i) => (
```

Comment:
Not a huge deal, but note that every render (i.e. any time anything changes) will call `this.statuses()` which will inspect the path and manipulate the status array, even though it never changes.

I'd prefer to do this once, say in `componentDidMount` and set a member variable, `this.statuses` which can be used in the render.

Reply:
good point! I will make that change.